crates/viewer/re_view_text_log/src/view_class.rs

```diff
@@ -438,6 +444,131 @@ fn table_ui(
     }
 }
 
+fn column_name_ui(
+    ctx: &ViewerContext<'_>,
+    ui: &mut egui::Ui,
+    column: &bp_datatypes::TextLogColumnKind,
+    timeline: &TimelineName,
+) -> egui::Response {
+    match column {
+        bp_datatypes::TextLogColumnKind::Timeline => item_ui::timeline_button(ctx, ui, timeline),
+        _ => ui.strong(column.name()),
+    }
+}
+
+// We need this to be a custom ui to bew able to use the view state to get seen text log levels. This could potentially be avoided if we could add component ui's from this crate.
```

Comment:
```suggestion
// We need this to be a custom ui to be able to use the view state to get seen text log levels.
// This could potentially be avoided if we could add component ui's from this crate.
```

well. so.. we actually can add component uis from this crate. Would be a first, but don't think there's anything to prevent us doing that in on_register